### DISCUSSION TITLE
[BUG] Password store says "No keys imported" even though my key is in OpenKeychain

### DISCUSSION THREAD

**Discussion by strikesmeasodd** [zeapo/Android-Password-Store#3219] [Q&A]
### Describe the bug

When I try to decrypt a password, the app says "No keys imported" and "There are no PGP keys imported in the app yet, press the button below to pick a key file".

OpenKeychain has the key and says it is Healthy and can Confirm other keys, Sign messages, and Decrypt messages.

If I try to import my key file through the dialog, it says "Found .gpg-id, but it contains an invalid key ID, fingerprint or user ID." 

I'm using the latest snapshot build  APS-freeRelease-2.0.0-SNAPSHOT.apk . I previously used the F-Droid release with no problems.

### Steps to reproduce

1. Import a key into OpenKeychain
2. Try to decrypt a key in Password-Store
3. See error

### Expected behavior

Password-Store should automatically detect my key in OpenKeychain, or else correctly import the key from a file.

### Device information

- Device: Samsung Galaxy A15
- OS: Android 14
- App version: Free release 2.0.0 snapshot from revision [341c5fa](https://github.com/android-password-store/Android-Password-Store/commit/341c5faac1a431d3345cb1b8cfc2e18dc724e849)
- OpenKeychain version 6.0.4

**Comment by msfjarvis** [comment]
The snapshot builds do not use OpenKeychain, see https://github.com/android-password-store/Android-Password-Store/issues/1523 for details.

You'll have to import your key in Password Store itself via Settings > PGP settings > Key manager
